doc/server.md

```diff
@@ -138,6 +138,7 @@ Key                                 | Type      | Scope     | Default
 :--                                 | :---      | :----     | :------                                          | :----------
 `backups.compression_algorithm`     | string    | global    | `gzip`                                           | Compression algorithm to use for new images (`bzip2`, `gzip`, `lzma`, `xz` or `none`)
 `instances.nic.host_name`           | string    | global    | `random`                                         | If set to `random`, use the random host interface name as the host name; if set to `mac`, generate a host name in the form `lxd<mac_address>` (MAC without leading two digits)
+`instances.placement_scriptlet`           | string    | global    | -                                          | Stores the {ref}`clustering-instance-placement-scriptlet` for custom automatic instance placement logic
```

Comment:
Is this formatted correctly? The diff looks a bit weird.

Reply:
Yep spacing not quite right